newbank/server/NewBank.java

```diff
@@ -150,6 +165,39 @@ private String showMyAccounts(CustomerID customer) {
 		return account.accountsToString();
 	}
 
+	private boolean addMoneyToAccount(CustomerID customerId, String accountName, String ammount) {
+		Double amount_to_num;
+
+		// TODO: Abstract this, duplicated from MOVE_MONEY method
+		try {
+			amount_to_num = Double.parseDouble(amount);
+
+			if (amount_to_num < 0) {
+				System.out.println("Invalid amount entered. Amount should be larger than 0.\n");
+				return false;
+			}
+			// Check if the amount is up to 2 decimal places
+			if ((BigDecimal.valueOf(amount_to_num).scale() > 2)) {
+				System.out.println("Invalid amount entered, give your amount to 2 decimal places.\n");
+				return false;
+			}
+		} catch (NumberFormatException e) {
+			System.out.println("Invalid amount entered.\n");
+			return false;
+		}
+
+		Customer customer = customers.get(customerId.getKey()).getCustomerValue();
+
+		for (Account a : customer.getAccounts()) {
+			if (a.getAccountName().equals(accountName)) {
+				a.credit_balance(amount_to_num);
```

Comment:
This should be using debit_balance (adding money to acc) instead of credit (deducting money from acc)